src/components/ContentHighlights/ContentHighlightCardItem.jsx

```diff
@@ -19,14 +19,24 @@ const ContentHighlightCardItem = ({
     cardImgSrc: cardImageUrl,
     cardLogoSrc: partners.length === 1 ? partners[0].logoImageUrl : undefined,
     cardLogoAlt: partners.length === 1 ? `${partners[0].name}'s logo` : undefined,
-    cardTitle: <Truncate lines={3} title={title}>{title}</Truncate>,
+    cardTitle: <LinesEllipsis
```

Comment:
[curious] Why the change from `react-truncate`? 

Could Paragon's `Truncate` component work here? See documentation: https://paragon-openedx.netlify.app/components/truncate/